pkg/parse/parse.go

```diff
@@ -208,19 +213,34 @@ func CommonBuildOptionsFromFlagSet(flags *pflag.FlagSet, findFlagFunc func(name
 
 // GetAdditionalBuildContext consumes raw string and returns parsed AdditionalBuildContext
 func GetAdditionalBuildContext(value string) (define.AdditionalBuildContext, error) {
```

Comment:
Locally, it’s not obvious that the checks for `""` are important; probably worth a more detailed comment.

Reply:
Adding a note there.